client/src/components/Modal/Modal.tsx

```diff
@@ -0,0 +1,62 @@
+import { FunctionComponent } from "react";
+import ReactDOM from "react-dom";
+import { Button } from "../Button";
+import "./style.scss";
+import { ModalProps } from "./types";
+
+const portalDiv = document.getElementById("portal") as HTMLElement;
+
+export const Modal: FunctionComponent<ModalProps> = ({
+  idForm,
+  onCancel,
+  open,
+  children,
+  heading,
+  buttonTextConfirm,
```

Comment:
children is presented in FunctionComponent type, all buttons props could be grouped into one object